docs/messaging/ios-configure-apns.md

```diff
@@ -5,37 +5,117 @@ description: Create an Apple Push Notification authentication key, a provisionin
 
 # Configure APNs for FCM
 
-## Before you begin
+### Setup Certificates
 
-This guide makes the following assumptions:
+#### Step 1:
 
-- you have an active [Apple Developer account](https://developer.apple.com/membercenter/index.action)
-- you have an active Firebase project
+Navigate to Certificates, Identifiers and Profiles.
 
-### Create an Authentication Key
+![Step 1](https://images.prismic.io/invertase/b498548c-de02-42bd-a7cf-e2b9a78f3803_FCM+-+step+1.png?auto=compress,format)
 
-<!-- https://firebase.google.com/docs/cloud-messaging/ios/certs -->
+#### Step 2:
 
-TODO
+Navigate to keys & click the `plus` button to create a new key.
 
-### Upload your APNs authentication key
+![Step 2](https://images.prismic.io/invertase/784f9941-efb2-4dd8-b74c-bd40f7ea0b0b_FCM+-+step+2.png?auto=compress,format)
 
-<!-- https://firebase.google.com/docs/cloud-messaging/ios/client#upload_your_apns_authentication_key -->
+#### Step 3:
 
-TODO
+Enter a name for your key & ensure you've enabled `Apple Push Notiffications service (APNs)`.
+
+![Step 3](https://images.prismic.io/invertase/8e1ba901-ec90-43d2-ae14-3681c9030610_FCM+-+step+3.png?auto=compress,format)
+
+#### Step 4:
+
+Register your key, ensure you download it & keep safe.
+
+![Step 4](https://images.prismic.io/invertase/ac5070c4-87ec-427a-ba6d-590f5feddbc8_FCM+-+step+4.png?auto=compress,format)
+
+#### Step 5:
+
+Register your key, ensure you download it & keep safe.
+
+![Step 5](https://images.prismic.io/invertase/ac5070c4-87ec-427a-ba6d-590f5feddbc8_FCM+-+step+4.png?auto=compress,format)
+
+
+Congratulations. You've setup the authentication key necessary for push notifications. You now need to create an App ID for your app via the Apple developer console.
+
+###Create an App ID
```

Comment:
This header also exists on line 79, duplicate?

Reply:
@Salakar Good catch, Mike. I've updated. Let me know if you want merging  👍